### PR TITLE
DEV-448 - Urgent Bug - system down

### PR DESCRIPTION
**Pull request by randyhoulahan**
- form malfunctioning because of multiple future conferences

### CODE REVIEW

File: app/directives/forms/edit/edit-side-event.html [hunk 1]
```diff
@@ -854,6 +854,7 @@ <h2>Responsible Person </h2>
                       <span ng-show="(doc.meta.status=== 'published' || doc.meta.status=== 'scheduled')">Edit Your Live Side-event</span>
                     </button>
                 </div>
+<pre>{{doc|json}}</pre>
```
Comment (blaisefonseca) [inline app/directives/forms/edit/edit-side-event.html]: remove debug

File: app/directives/forms/edit/edit-side-event.js [hunk 2]
```diff
@@ -163,8 +165,10 @@ define(['app', 'lodash',
                         //============================================================
                         function loadConferences() {
                             return $http.get('/api/v2016/conferences', { 'params': findOpenRegsQuery() }).then(function(res) {
+                                const conferences = res.data.sort(compareDates).reverse();
+                                const { meetingId } = $route.current.params || {};
 
-                                $scope.options.conferences=res.data.sort(compareDates);
+                                $scope.options.conferences = isNeww() && meetingId? [_.find(conferences, { MajorEventIDs: [ meetingId ] })] : res.data.sort(compareDates).reverse();
```
Comment (blaisefonseca) [inline app/directives/forms/edit/edit-side-event.js]: `isNeww` correct spelling

File: app/directives/forms/edit/edit-side-event.html (hunk 1 above)
Comment (randyhoulahan) [reply app/directives/forms/edit/edit-side-event.html]: @blaisefonseca resolved.  thanks for the catch

File: app/directives/forms/edit/edit-side-event.js (hunk 2 above)
Comment (randyhoulahan) [reply app/directives/forms/edit/edit-side-event.js]: yes I saw, thanks for calling out my laziness :-)

**Review by blaisefonseca — APPROVED** [review]
there was a json

File: app/directives/forms/edit/edit-side-event.js (hunk 2 above)
Comment (randyhoulahan) [reply app/directives/forms/edit/edit-side-event.js]: @blaisefonseca resolved

**Pull request merged**